src/main/java/uk/gov/hmcts/reform/juddata/camel/processor/JudicialOfficeAuthorisationProcessor.java

```diff
@@ -68,6 +66,9 @@ public void process(Exchange exchange) {
             setFileStatus(exchange, applicationContext);
         }
 
+        log.info("{}:: Judicial Authorisation Records count after Validation {}:: ", logComponentName,
+            filteredJudicialAuthorisations.size());
```

Comment:
not sure why did we move the log below audit?
if there is any exception in audit then the log will not be printed...